presto-orc/src/main/java/com/facebook/presto/orc/reader/ListStreamReader.java

```diff
@@ -63,19 +76,32 @@
     private int nextBatchSize;
 
     private InputStreamSource<BooleanInputStream> presentStreamSource = missingStreamSource(BooleanInputStream.class);
```

Comment:
Prior comment:

---
presentStreamSource variable is defined in ColumnReader; remove from here